base/src/utils.rs

```diff
@@ -367,3 +367,40 @@ mod tests {
         assert!(!is_valid_hex_color("#ffffff00")); // with alpha channel
     }
 }
+
+pub(super) fn get_column_number(s: &str, str_len: usize) -> u32 {
```

Comment:
These functions are already implemented elsewhere. Search for instance `number_to_column`

Reply:
These are cleaned up, as parse_range itself will take care of these things